### DISCUSSION TITLE
为什么不能打包Windows版本了

### DISCUSSION THREAD

**Discussion by lqingjin** [tw93/Pake#364] [Q&A]
最近打包不能打包win版本了，只有mac和Linux版，请问是什么原因呢，在线求解

**Comment by tw93** [comment]
你说的 pake 官方常用包里面的吗？我重新触发一下应该就好了

**Reply by zengqingrong** [reply]
hello，老哥，看起来还是不行，常用包里面的链接挂了，我自己跑Action也没有附件输出

**Reply by Tlntin** [reply]
你拉一下最新代码再去跑actions就可以了。

**Reply by pronouncing** [reply]
同样的问题，通过actions方式打包一个网页，Windows打包成功，但是没有附件下载，不知道什么原因

**Comment by Tlntin** [comment]
这个问题是因为上传的那个脚本更新后不支持windows了，两天前我就修复了，并且手动打包（`Build App with Cache`）也支持自定义版本号。不过没有触发重新打包而已，刚刚我已经重新触发了，

**Reply by lqingjin** [reply]
好的，我再试试，谢谢

**Reply by pronouncing** [reply]
同样的问题，通过actions方式打包一个网页，Windows打包成功，但是没有附件下载，不知道什么原因

**Reply by Tlntin** [reply]
> 同样的问题，通过actions方式打包一个网页，Windows打包成功，但是没有附件下载，不知道什么原因

我看你的日志是有附件的哈。

**Reply by pronouncing** [reply]
我删除了缓存，重新构建了一下，后面成功编译，附件才有了。

**Reply by pronouncing** [reply]
还有请教一个问题能打包成安卓APP吗

**Reply by Tlntin** [reply]
> 还有请教一个问题能打包成安卓APP吗

目前还不行，tauri官方正在支持中，后续正式支持了会发教程帖子。
不过体验了一下开发安卓比较复杂，需要安卓的开发工具(安卓studio),sdk, ndk，还需要安装微软开发的C++依赖跨平台管理工具vcgk。
总的来说，大概是目前开发环境搭建难度的5-10倍。
